SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebugger.h

```diff
@@ -258,4 +307,31 @@ class SPATIALGDK_API ASpatialDebugger : public AInfo
 	FCanvasIcon Icons[ICON_MAX];
 
 	USpatialDebuggerConfigUI* ConfigUIWidget;
+
+	// Mode for selecting actors under the cursor - should only be visible in the runtime config UI
+	bool bSelectActor = false;
+
+	// Actors selected by user for debugging
+	TArray<TWeakObjectPtr<AActor>> SelectedActors;
+
+	// Highlighted actor under the mouse cursor
+	TWeakObjectPtr<AActor> HoverActor;
+	// Highlighted actor original materials and components
+	TArray<TWeakObjectPtr<UMaterialInterface>> ActorMeshMaterials;
+	TArray<TWeakObjectPtr<UMeshComponent>> ActorMeshComponents;
+	// Material for highlighting actor
+	UPROPERTY()
+	UMaterialInterface* WireFrameMaterial;
+
+	// All actors under the mouse cursor
+	TArray<TWeakObjectPtr<AActor>> HitActors;
+
+	// Index for selecting the highlighted actor when multiple are under the mouse cursor
+	uint64 HoverIndex;
```

Comment:
64 bit int seems a little excessive here, is there a max return num of actors returned for hit detection?

Reply:
Yes, you are right changing to a 32 int as it is sufficient.